lib/addons/amp/index.ts

```diff
@@ -77,14 +107,23 @@ export class AmpAddOn implements ClusterAddOn {
             doc = readYamlDocument(__dirname + '/collector-config-amp.ytpl');
         }
 
-        const manifest = doc.split("---").map(e => loadYaml(e));
+        const manifest = doc.split("---").map(e => {
+            let object = loadYaml(e);
+
+            if (this.ampAddOnProps.openTelemetryCollectorManifestPath !== undefined && object.kind === "OpenTelemetryCollector"){
```

Comment:
The intention here is to overwrite the yaml doc from  `collector-config-amp.ytpl` to `openTelemetryCollectorManifestPath` is passed?

Reply:
correct (or collector-config-amp-daemonset.ytpl)